ci_scripts/ci-prelude.sh

```diff
@@ -46,16 +46,28 @@ case $TRAVIS_OS_NAME in
         fi
         ;;
     osx)
-        sudo pip3 install meson
-        ./scripts/build_sasl.sh --with-openssl=/usr/local/opt/openssl
+        # Install meson, ninja, openssl & libetpan
+        brew install openssl meson
+        brew upgrade libetpan
```

Comment:
These do not do the same as the `packages` section in `.travis.yml`?  I mean e.g. openssl isn't already brew installed and that the upgrade of libetpan changes anything.  Possibly some travis magic which i misjudged though.

Reply:
oh, probably something to do with the brew cleanup bit you added?

Reply:
might be worth removing them from travis.yml to avoid confusion about which thing is being used.

Reply:
> These do not do the same as the `packages` section in `.travis.yml`? I mean e.g. openssl isn't already brew installed and that the upgrade of libetpan changes anything. Possibly some travis magic which i misjudged though.

Oooh good point. Yes they do. brew cleanup is only for the caching, will modify that.